### PR TITLE
Added some highlights for .NET MAUI

### PR DESCRIPTION
**Pull request by davidortinau**
---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/whats-new/dotnet-9/overview.md](https://github.com/dotnet/docs/blob/d70d9783ff2a223e6e55ba47a7b9e0f9a8d506e1/docs/core/whats-new/dotnet-9/overview.md) | [What's new in .NET 9](https://review.learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-9/overview?branch=pr-en-us-43411) |

### CODE REVIEW

**Review by jamesmontemagno — APPROVED** [review]

File: docs/core/whats-new/dotnet-9/overview.md [hunk 1]
```diff
@@ -67,7 +67,15 @@ ASP.NET Core includes improvements to Blazor, SignalR, minimal APIs, OpenAPI, an
 
 ## .NET MAUI
 
-The focus of .NET Multi-platform App UI (.NET MAUI) in .NET 9 is to improve product quality. For more information about that and new features, see [What's new in .NET MAUI for .NET 9](/dotnet/maui/whats-new/dotnet-9).
+The focus of .NET Multi-platform App UI (.NET MAUI) in .NET 9 is to improve product quality. In addition:
```
Comment (jamesmontemagno) [inline docs/core/whats-new/dotnet-9/overview.md:70]:
```suggestion
The focus of .NET Multi-platform App UI (.NET MAUI) in .NET 9 is focused on enhanced performance, reliability, and deeper integrations for desktop and mobile applications. .NET MAUI includes a new more performant implementation of `CollectionView` & `CarouselView` for iOS & Mac Catalyst, updates to existing controls, new app lifecycle events, and Native AOT and trimming enhancements to improve app size and startup time.  In addition:
```

File: docs/core/whats-new/dotnet-9/overview.md [hunk 2]
```diff
@@ -67,7 +67,15 @@ ASP.NET Core includes improvements to Blazor, SignalR, minimal APIs, OpenAPI, an
...
+The focus of .NET Multi-platform App UI (.NET MAUI) in .NET 9 is enhanced performance, reliability, and deeper integrations for desktop and mobile applications. .NET MAUI includes a new more performant implementation of `CollectionView` & `CarouselView` for iOS & Mac Catalyst, updates to existing controls, new app lifecycle events, and Native AOT and trimming enhancements to improve app size and startup time.  In addition:
+
+- A new `TitleBar` desktop control is available for Windows
+- A new `HybridWebView` control enables easier inclusion of JavaScript enabled content from frameworks like ReactJS, Vue.js, and Angular
+- `Entry` now supports additional keyboard modes
+- Control handlers automatically disconnect from their controls when possible
+- `MainPage` is deprecated in favor of setting the primary page of the app by overriding `CreateWindow` in the `App` class
+  
+For more information about that and new features, see [What's new in .NET MAUI for .NET 9](/dotnet/maui/whats-new/dotnet-9).
```
Comment (jamesmontemagno) [inline docs/core/whats-new/dotnet-9/overview.md:78]:
```suggestion
For more information about that these new features and more, see [What's new in .NET MAUI for .NET 9](/dotnet/maui/whats-new/dotnet-9).
```

**Review by jamesmontemagno — APPROVED** [review]
Added a bit more ;)

File: docs/core/whats-new/dotnet-9/overview.md [hunk 3]
```diff
@@ -67,7 +67,15 @@ ASP.NET Core includes improvements to Blazor, SignalR, minimal APIs, OpenAPI, an
 
 ## .NET MAUI
 
-The focus of .NET Multi-platform App UI (.NET MAUI) in .NET 9 is to improve product quality. For more information about that and new features, see [What's new in .NET MAUI for .NET 9](/dotnet/maui/whats-new/dotnet-9).
+The focus of .NET Multi-platform App UI (.NET MAUI) in .NET 9 is enhanced performance, reliability, and deeper integrations for desktop and mobile applications. .NET MAUI includes a new more performant implementation of `CollectionView` & `CarouselView` for iOS & Mac Catalyst, updates to existing controls, new app lifecycle events, and Native AOT and trimming enhancements to improve app size and startup time.  In addition:
```
Comment (gewarren) [inline docs/core/whats-new/dotnet-9/overview.md:70]:
```suggestion
The focus of .NET Multi-platform App UI (.NET MAUI) in .NET 9 is enhanced performance and reliability, and deeper integrations for desktop and mobile applications. .NET MAUI includes a new, more performant implementation of <xref:Microsoft.Maui.Controls.CollectionView> and <xref:Microsoft.Maui.Controls.CarouselView> for iOS and Mac Catalyst, updates to existing controls, new app lifecycle events, and Native AOT and trimming enhancements to improve app size and startup time.  In addition:
```

File: docs/core/whats-new/dotnet-9/overview.md [hunk 4]
```diff
@@ -67,7 +67,15 @@ ASP.NET Core includes improvements to Blazor, SignalR, minimal APIs, OpenAPI, an
...
+
+- A new `TitleBar` desktop control is available for Windows
+- A new `HybridWebView` control enables easier inclusion of JavaScript enabled content from frameworks like ReactJS, Vue.js, and Angular
+- `Entry` now supports additional keyboard modes
+- Control handlers automatically disconnect from their controls when possible
+- `MainPage` is deprecated in favor of setting the primary page of the app by overriding `CreateWindow` in the `App` class
```
Comment (gewarren) [inline docs/core/whats-new/dotnet-9/overview.md:76]:
```suggestion
- A new <xref:Microsoft.Maui.Controls.TitleBar> desktop control is available for Windows.
- A new <xref:Microsoft.Maui.Controls.HybridWebView> control enables easier inclusion of JavaScript-enabled content from frameworks like ReactJS, Vue.js, and Angular.
- <xref:Microsoft.Maui.Controls.Entry> now supports additional keyboard modes.
- Control handlers automatically disconnect from their controls when possible.
- <xref:Microsoft.Maui.Controls.Application.MainPage> is deprecated in favor of setting the primary page of the app by overriding <xref:Microsoft.Maui.Controls.Application.CreateWindow(Microsoft.Maui.IActivationState)?displayProperty=nameWithType> class.
```

**Review by gewarren — APPROVED** [review]

**Pull request merged**